docker-paradox.toml

```diff
@@ -0,0 +1,11 @@
+[general]
+port = 3030
+public = false
+domain = "localhost"
```

Comment:
This should be a docker variable. It's not actually used to bind but used for CORS and friends

Reply:
Cool deal, I was wondering what it was used for, i just started digging into the code.
Will update and test